solutions/instance/variables.tf

```diff
@@ -105,7 +105,7 @@ variable "service_credential_secrets" {
   }
 }
 
-variable "skip_cos_sm_auth_policy" {
+variable "skip_cos_secrets_manager_auth_policy" {
```

Comment:
In secrets the input is called `skip_kms_iam_authorization_policy`. So we are not consistent with naming convention here